src/api/app/controllers/webui/webui_controller.rb

```diff
@@ -18,6 +18,7 @@ class Webui::WebuiController < ActionController::Base
   before_action :setup_view_path
   before_action :check_user
   before_action :check_anonymous
+  before_action :check_spider
```

Comment:
```suggestion
  before_action :check_spider
  before_action :check_anonymous
```
Otherwise `@spider_bot` won't be set, and therefore won't be tracked, for pages requiring authentication...